Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -169,7 +199,7 @@ public override void Update(float deltaTime, Camera cam)
                     CancelUsing(user);
                     user = null;
                 }
-                if (!IsToggle || item.Connections == null) { IsActive = false; }
+                if (item.Connections == null) { IsActive = false; }
```

Comment:
Why was the `!IsToggle` check removed? I don't see why we should not deactivate a non-toggleable component here.

Reply:
Readded the check in commit [af157a7](https://github.com/Regalis11/Barotrauma/pull/10419/commits/af157a79551d2add3ae4bbc5050212d39b18e77d).